contrib/fsl_integration/sct_plugin.py

```diff
@@ -191,8 +249,8 @@ def onButtonSC(self, event):
         opts.cmap = 'red'
 
 
+#Run Gray Matter Segmentation of Spinal Cord
```

Comment:
the output of GM seg in fsleyes is "seg" instead of "t2s_seg". It would be nice to fix it.